.github/workflows/ci.yaml

```diff
@@ -47,11 +47,9 @@ jobs:
         uses: cachix/install-nix-action@v22
         with:
           github_access_token: ${{ secrets.GITHUB_TOKEN }}
-          nix_path: nixpkgs=channel:nixos-unstable
-      - name: Install Rust
-        uses: dtolnay/rust-toolchain@master
-        with:
-          toolchain: ${{ matrix.channel }}
+          nix_path: nixpkgs=channel:nixpkgs-unstable
+      - name: Enter devshell
+        run: nix develop .#full
```

Comment:
Each step is run in a separated environment so this does nothing and exits immediately. You need `nix develop .#full --command XXX`.

Reply:
ah should be fixed